py-pkg/pkgnet/module_reporter.py

```diff
@@ -0,0 +1,45 @@
+import pandas as pd
+
+from pkgnet.abstract_graph_reporter import AbstractGraphReporter
+from pkgnet.graphs import DirectedGraph
+from pkgnet.package_report import register_reporter
+from pkgnet.search_utils import get_all_modules_in_package, get_submodules
+
+
+@register_reporter
+class ModuleReporter(AbstractGraphReporter):
```

Comment:
Recently been thinking that maybe `SubmoduleReporter` would be a better name.

Reply:
I like `SubmoduleReporter`! Ok now I'm really curious to see what this looks like on a package with a gagillion submodules `networkx`